static/app/components/devtoolbar/components/alerts/alertsPanel.tsx

```diff
@@ -0,0 +1,185 @@
+import {css} from '@emotion/react';
+
+import ActorAvatar from 'sentry/components/avatar/actorAvatar';
+import AlertBadge from 'sentry/components/badge/alertBadge';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import TimeSince from 'sentry/components/timeSince';
+import type {Actor} from 'sentry/types/core';
+import type {Organization} from 'sentry/types/organization';
+import AlertRuleStatus from 'sentry/views/alerts/list/rules/alertRuleStatus';
+import type {Incident, MetricAlert} from 'sentry/views/alerts/types';
+import {CombinedAlertType} from 'sentry/views/alerts/types';
+import {alertDetailsLink} from 'sentry/views/alerts/utils';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import {
+  badgeWithLabelCss,
+  gridFlexEndCss,
+  listItemGridCss,
+  listItemPlaceholderWrapperCss,
+} from '../../styles/listItem';
+import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {resetFlexColumnCss, resetFlexRowCss} from '../../styles/reset';
+import {smallCss, xSmallCss} from '../../styles/typography';
+import InfiniteListItems from '../infiniteListItems';
+import InfiniteListState from '../infiniteListState';
+import PanelLayout from '../panelLayout';
+import SentryAppLink from '../sentryAppLink';
+import useTeams from '../teams/useTeams';
+
+import useInfiniteAlertsList from './useInfiniteAlertsList';
+
+export default function AlertsPanel() {
+  const {projectId, projectSlug, trackAnalytics} = useConfiguration();
+  const queryResult = useInfiniteAlertsList();
+
+  const estimateSize = 108;
+  const placeholderHeight = `${estimateSize - 8}px`; // The real height of the items, minus the padding-block value
```

Comment:
should we make these constants/export them somewhere? they're used in the feedback & issues panels too

Reply:
oops, these should probably be unique values